app/utils/initializer.js

```diff
@@ -1,10 +1,28 @@
 require('../config');
-require('../services/elasticSearch');
 
-const loggerInitialization = () => Promise.resolve();
+const logger = require('../utils/logger')('initializer');
+const { elasticSearchClient } = require('../services/elasticSearch');
+
+const elasticSearchInitialization = () => new Promise((resolve, reject) => {
+  elasticSearchClient.ping({
```

Comment:
I think, you need to `return` this method

Reply:
Why?